app/models/concerns/user_reports.rb

```diff
@@ -0,0 +1,20 @@
+module UserReports
```

Comment:
Not sure about the module name here - it seems like its about the ability to manager reports about users in my mind! I assume this module is to determine access to Reporting features - can it go in the existing `UserRole` module rather than a separate place?